osbrain/agent.py

```diff
@@ -898,6 +898,44 @@ def _connect_and_register(self, client_address, alias=None, handler=None,
         self.register(socket, register_as, alias, handler)
         return client_address
 
+    def unsubscribe_socket_from_topic(self, socket, topic: bytes):
```

Comment:
Maybe `unsubscribe_from_topic()` would be better. First parameter could be an AgentAddress, an alias...

Also, if you set the `topic` type (i.e.: `topic: bytest`), do the same for all parameters. :wink: